test/system/comment_test.rb

```diff
@@ -38,17 +39,6 @@ def get_path(page_type, path)
       assert_selector('#comments-list .comment-body p', text: comment_text)
     end
 
-    test "#{page_type_string}: reply to existing comment" do
-      visit get_path(page_type, nodes(node_name).path)
-      # find comment ID of the first comment on page
-      parent_id = "#" + page.find('#comments-list').first('.comment')[:id]
-      parent_id_num = /c(\d+)/.match(parent_id)[1] # eg. comment ID format is id="c9834"
-      # addComment(comment text, submitURL, comment's parent ID)
-      page.evaluate_script("addComment(\"no you can't\", '/comment/create/#{nodes(:comment_note).nid}', #{parent_id_num})")
-      # check for comment text
-      assert_selector("#{parent_id} .comment .comment-body p", text: 'no you can\'t')
-    end
-
```

Comment:
I deleted this test (which I wrote early on in the internship), because it doesn't seem so necessary now.